lua/neogit/popups/init.lua

```diff
@@ -44,8 +43,7 @@ function M.mappings_table()
       "CommitPopup",
       "Commit",
       M.open("commit", function(f)
-        local commit = require("neogit.status").get_selected_commits()[1]
-        f { commit = commit and commit.oid }
+        f { selection = require("neogit.status").get_selection() }
```

Comment:
No objections to this change, just remember that the log buffer and reflog buffer are still passing in a table of oid strings to these popup constructors.